external/source/test/README.md

```diff
@@ -111,79 +105,36 @@ You can set these environment variables to configure the test suite:
     TEST_SHOW_BROWSER  Set to run browser interactively. When not specified,
                        browser is run in headless mode.
 
-In addition, you can also set the `cockpit.bots.images-data-dir` variable with
-`git config` to the location to store the (unprepared) virtual machine images.
-This takes precedence over `TEST_DATA`.  For example:
-
-    $ git config cockpit.bots.images-data-dir ~/.cockpit-bots/images
-
 ## Test machines and their images
 
 The code under test is executed in one or more dedicated virtual
 machines, called the "test machines".  Fresh test machines are started
-for each test. To pull all the needed images for a given commit, use:
-
-    $ bots/image-download
-
-A test machine runs a "test machine image".  Such a test machine image
-contains the root filesystem that is booted inside the virtual
-machine.  A running test machine can write to its root filesystem, of
-course, but these changes are (usually) not propagated back to its
-image.  Thus, you can run multiple test machines from the same image,
-at the same time or one after the other, and each test machine starts
-with a clean state.
-
-A test machine image is created with image-create, like so:
-
-    $ bots/image-create -v fedora-testing
-
-The image will be created in `$TEST_DATA/images/`. In addition a link
-reference will be created in `bots/images/`.
-
-If you wish that others use this new image then you should commit the
-new reference link, and use `image-upload` to upload the new image. You
-would need to have Cockpit commit access to do this:
-
-    $ bots/image-upload fedora-testing
-
-There is more than one test machine image. For example, you might
-want to test a scenario where Cockpit on one machine talks to FreeIPA
-on another, and you want those two machines to use different images.
-
-This is handled by passing a specific image to image-create
-and other scripts that work with test machine images.
-
-    "fedora-NN" -- The basic image for running the development version of Cockpit.
-                   This is the default.
-
-    "ipa"       -- A FreeIPA server.
-
-    "openshift" -- An Openshift Origin server.
+for each test. See the
+[bots documentation](https://github.com/cockpit-project/bots/blob/master/README.md)
```

Comment:
Oh, I need to add a rewrite to keep the bots link on our site, instead of going to GitHub.

Reply:
Why does that not work? The commit doesn't really explain why this now pulls in bots README. Just for consistently looking formatting?